network/p2p/network.go

```diff
@@ -545,3 +533,19 @@ func (n *Network) queueSubmitFunc(message interface{}) {
 
 	n.metrics.InboundProcessDuration(qm.Target.String(), time.Since(startTimestamp))
 }
+
+func EventId(channel network.Channel, payload []byte) (hash.Hash, error) {
+	// use a hash with an engine-specific salt to get the payload hash
+	h := hash.NewSHA3_384()
+	_, err := h.Write([]byte("libp2ppacking" + channel))
```

Comment:
```suggestion
        const prefix = "libp2ppacking"
	_, err := h.Write([]byte(preifx + channel))
```

Reply:
maybe golang is a lil more efficient with constants than a string being define on each method call

Reply:
https://github.com/onflow/flow-go/pull/1908/commits/603f57f7ae1f2bc17bc7fc61d55375d501b6dc32